notebooks/delta_table_upload_demo.ipynb

```diff
@@ -0,0 +1,492 @@
+      "source": [
+        "Install the required dependencies if you are uploading data from a Delta table stored in a third-party cloud service.\n",
+        "\n",
+        "AWS S3 dependencies:"
+      ],
+      "metadata": {
+        "id": "RMXEYd4FTQeW"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!wget -P /usr/local/lib/python3.10/dist-packages/pyspark/jars https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.4/hadoop-aws-3.3.4.jar\n",
+        "!wget -P /usr/local/lib/python3.10/dist-packages/pyspark/jars https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.12.262/aws-java-sdk-bundle-1.12.262.jar"
+      ],
+      "metadata": {
+        "id": "4-ICLmvn5mSI"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "GCS dependencies:"
+      ],
+      "metadata": {
+        "id": "fncOoUfm0g1n"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!wget -P /usr/local/lib/python3.10/dist-packages/pyspark/jars https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop3-latest.jar"
+      ],
+      "metadata": {
+        "id": "gVFX2uY69u4n"
+      },
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The spark_config argument is only required when connecting to a third-party storage service such as AWS S3 or GCS, the fields that need to be in spark_config are:\n",
+        "\n",
+        "\n",
+        "*   jars - A string containing the path to the .jar files needed for the connector you are using, separated by a comma. The .jar file versions will need to be compatible with the version of Spark you have installed.\n",
+        "\n",
+        "  *   AWS S3 - hadoop-aws-2.7.4.jar, aws-java-sdk-1.7.4.jar\n",
+        "  *   GCS - gcs-connector-hadoop3-latest.jar\n",
+        "*   credentials - Only required if connecting to GCS, the value should be the path to your [GCS service account](https://developers.google.com/workspace/guides/create-credentials#create_credentials_for_a_service_account) credentials JSON file. To run this notebook with a GCS credentials file, click on the folder icon on the left sidebar and then click on the upload file icon. This will upload the file to the Notebook's `/content/` folder.\n",
+        "\n",
+        "\n",
+        "*   AWS_ACCCESS_KEY - Only required if connecting to AWS S3, the value should be your [AWS service account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) access key\n",
+        "*   AWS_SECRET_KEY - Only required if connecting to AWS S3, the value should be your [AWS service account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) secret key\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "y8b9Awq7YAfg"
+      }
+    },
+    {
+      "cell_type": "code",
```

Comment:
I would switch this table with the data table creation cell, I think it makes more sense to assume the customer is running this for the first time and don't have a table ready to test

Reply:
Good call, moved this cell and the two following cells just below the import cell